bika/lims/api/__init__.py

```diff
@@ -1265,6 +1266,33 @@ def to_date(value, default=None):
         return to_date(default)
 
 
+def to_minutes(days=0, hours=0, minutes=0, seconds=0, milliseconds=0,
+               to_int=True):
```

Comment:
I would leave in general the type conversion to the calling function and return only a string here.
Also that `to_int` does actually also a `round` which is not directly obvious to the caller

Reply:
Umm, don't like to do `int(round(api.get_minutes(**kwargs)))` everywhere. I suggest to change the param name from `to_int` to `round_to_int`